test/setup.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Microsoft Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+matrix({
+  'browserName': process.env.BROWSER ? [process.env.BROWSER] : ['chromium', 'webkit', 'firefox'],
+  'headless': [!!valueFromEnv('HEADLESS', true)],
+  'wire': [!!process.env.PWWIRE],
```

Comment:
Why don't we make this into actual parameter?

`process.env.PWWIRE ? [true] : [true, false]`

Reply:
Well, because you don't want to run both by default?